### DISCUSSION TITLE
Deeplinking

### DISCUSSION THREAD

**Discussion by riccardo-runci** [serverpod/serverpod#1837] [Q&A] [answered]
Hello, is possible with web server to create a deep linking to redirect the user to the app? for example with success url of stripe checkout

**Comment by vlidholt** [comment, marked as answer]
Yes, most definitely. You may want to use the webserver (Relic) for things like this. It's still under development, but you can find some info here:
https://docs.serverpod.dev/concepts/webserver

**Comment by beekboss** [comment]
@riccardo-runci  did you success in deep link?
